scripts/migration/migrate_deleted_wikis.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.WIKI_DELETED))
+
+def migrate(targets, dry_run=True):
+    # iterate over targets
+    for log in targets:
+        node = log.node
+        versions = node.wiki_pages_versions
+        current = node.wiki_pages_current
+        updated_versions = {}
+        for wiki in versions:
+            if wiki in current:
+                updated_versions[wiki] = versions[wiki]
+        with TokuTransaction():
```

Comment:
You also don't need a withTokuTransaction() in this function because the call to `migrate()` is wrapped in a transaction from the call in the main() function.